### DISCUSSION TITLE
Not compatible with Mac os Monterey

### DISCUSSION THREAD

**Discussion by BreakingVap** [kyleneideck/BackgroundMusic#555] [General]
Hello i have update my Mac OS to Monterey but BackgroundMusic crash.
I have try Big sur version but it don't work.
Have you got a solution ?

**Comment by edoardo-conti** [comment]
No update?

**Comment by Pepe424** [comment]
I'm currently running latest version of BGmusic built by my own on MBA M1 Monterey without any issues. I'm using it mostly for capturing internal sound in OBS.
<img width="816" alt="image" src="https://user-images.githubusercontent.com/79546326/159248140-d9d3fc45-336f-465e-a15e-571c290f6be4.png">

**Reply by edoardo-conti** [reply]
In the next few days I will try to compile it too and I'll let you know the result.

**Comment by haarismemon** [comment]
Hey @Pepe424, I'm quite new to this. How were you able to build your own version?

And hey @edoardo-conti, any luck on getting it to compile?

**Reply by Pepe424** [reply]
Just follow instructions Installing from Source Code in README.md file.

**Reply by haarismemon** [reply]
Thanks @Pepe424. I ran the build_and_install.sh script from the latest master but it fails to install the app on the 3rd step. The reason for the failure isn't super clear, any ideas why it could be failing? Here's a screenshot of the installation:

<img width="1440" alt="Screenshot 2022-04-02 at 16 35 49" src="https://user-images.githubusercontent.com/16806617/161390392-4588b6c8-0e82-4130-873e-30b3979beec7.png">

**Reply by Pepe424** [reply]
I know a had the same issue when try to built first time. Do you have Xcode app installed and opened for the first time?

**Reply by haarismemon** [reply]
Ahh yep that's what it was. I installed Xcode just for this and hadn't opened up the Xcode app for the first time. When I opened it up, it prompted me to install additional required components. I've now been able to run the install script and run BackgroundMusic on macOS monterey (12.3). Thanks @Pepe424!
